docs/concepts/scheduler.md

```diff
@@ -51,31 +55,61 @@ However, the Gardener Scheduler on the other hand does not need a TLS configurat
 The scheduling strategy is defined in the _**candidateDeterminationStrategy**_ of the scheduler's configuration and can have the possible values `SameRegion` and `MinimalDistance`.
 The `SameRegion` strategy is the default strategy.
 
-1. *Same Region strategy*
+### Same Region strategy
 
-   The Gardener Scheduler reads the `spec.provider.type` and `.spec.region` fields from the `Shoot` resource.
+The Gardener Scheduler reads the `spec.provider.type` and `.spec.region` fields from the `Shoot` resource.
 It tries to find a seed that has the identical `.spec.provider.type` and `.spec.provider.region` fields set.
 If it cannot find a suitable seed, it adds an event to the shoot stating that it is unschedulable.
 
-2. *Minimal Distance strategy*
-
-   The Gardener Scheduler tries to find a valid seed with minimal distance to the shoot's intended region.
-The distance is calculated based on the Levenshtein distance of the region. Therefore, the region name
+### Minimal Distance strategy
+
+The Gardener Scheduler tries to find a valid seed with minimal distance to the shoot's intended region.
+Distances are configured via `ConfigMap`(s), usually per cloud provider in a Gardener landscape.
+The configuration is structured as the following:
+- It refers to one or multiple `CloudProfile`s via annotation `scheduling.gardener.cloud/cloudprofiles`.
+- It contains the declaration as `region-config` via label `scheduling.gardener.cloud/purpose`.
+- If a `CloudProfile` is referred by multiple `ConfigMap`s, only the first one is considered.
+- The `data` fields configure actual distances, where _key_ relates to the `Shoot` region and _value_ contains distances to `Seed` regions. The region distance to itself needs to be specified as well.
```

Comment:
> The region distance to itself needs to be specified as well.

Should we change this and always assume weight `0` for the same region? At least our generated example files don't contain the same region in the values.